.ci/dockerfiles/Dockerfile.ctyunos23.01

```diff
@@ -0,0 +1,26 @@
+ARG ARCH=aarch64
+ARG HARBOR_URL=harbor.mellanox.com
+# FROM --platform=linux/{arm64|amd64} urm.nvidia.com/sw-nbu-build-docker/base/ctyunos:23.01
```

Comment:
Please clean all comments, we also need to have our own copy of the base image to avoid cases where the original image is moved/changed (you can add comment indicating where the original image is)

Reply:
Done